src/include/CMakeLists.txt

```diff
@@ -82,8 +82,7 @@ else ()
          ${CMAKE_BINARY_DIR}/include/OpenImageIO/detail/fmt/ostream.h
          ${CMAKE_BINARY_DIR}/include/OpenImageIO/detail/fmt/printf.h )
     foreach (f format.h ostream.h printf.h)
-        file (WRITE "${CMAKE_BINARY_DIR}/include/OpenImageIO/detail/fmt/${f}"
-                    "#include <fmt/${f}>")
+        configure_file ("${CMAKE_CURRENT_LIST_DIR}/OpenImageIO/${f}.in" "${CMAKE_BINARY_DIR}/include/OpenImageIO/detail/fmt/${f}" COPYONLY)
```

Comment:
If I understand correctly, you are using configure_file to hijack its ability to run only if the file is not already created, right? But doing so at the expense of adding three .in files and needing to transform them even though there is no actual configuration or variable substitution.

I wonder, then, if a simpler alternate approach might be to stick with `file(WRITE, ...)` but to surround it in an `if` that only does the write if the file doesn't already exist? Would that also solve the problem of not writing it unnecessarily (and therefore triggering recompiles), but without needing to add and transform the .in files?